### PR TITLE
add new Dexes

### PR DESCRIPTION
**Pull request by ComeOnOliver**
Add Support for Aftermath, Kriya V2/V3

### CODE REVIEW

File: src/lib/retrieveAPI.ts [hunk 1]
```diff
@@ -20,22 +20,26 @@ export async function getRoute(
     fromCoin: string,
     toCoin: string,
     amountIn: number | string | bigint,
-    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number } = { dexList: [Dex.Cetus], byAmountIn: true, depth: 3 }
+    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number } = { dexList: [], byAmountIn: true, depth: 3 }
```
Comment (alexayan) [inline src/lib/retrieveAPI.ts]: Define a new type ’SwapOptions‘

File: src/main.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export async function swapPTB(
     coin: TransactionResult,
     amountIn: number | string | bigint,
     minAmountOut: number,
-    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number } = { dexList: [Dex.Cetus], byAmountIn: true, depth: 3 }
+    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number } = { dexList: [], byAmountIn: true, depth: 3 }
```
Comment (alexayan) [inline src/main.ts]: use SwapOptions type

File: src/main.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ export async function swap(
     toCoin: string,
     amountIn: number | string | bigint,
     minAmountOut: number,
-    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number, isDryRun?: boolean, keypair?: Ed25519Keypair } = { dexList: [Dex.Cetus], byAmountIn: true, depth: 3, isDryRun: true, keypair: undefined }
+    swapOptions: { dexList?: Dex[], byAmountIn?: boolean, depth?: number, isDryRun?: boolean, keypair?: Ed25519Keypair } = { dexList: [], byAmountIn: true, depth: 3, isDryRun: true, keypair: undefined }
```
Comment (alexayan) [inline src/main.ts]: use SwapOptions type

**Review by alexayan — APPROVED** [review]

**Review by navi-dn — APPROVED** [review]
lgtm

**Review by navi-dn — APPROVED** [review]
lgtm

**Pull request merged**